hasher-matcher-actioner/hmalib/common/config.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Helpers for writing simple configs backed by a DynamoDB table
+
+Uses dataclass reflection to try and simplifying going between
+AWS API types and local types. There's likely already an existing
+library that exists somewhere that does this much better.
+"""
+from decimal import Decimal
+from dataclasses import dataclass, fields, is_dataclass
+import typing as t
+
+import boto3
+from boto3.dynamodb.conditions import Attr
+
+T = t.TypeVar("T")
+TConfig = t.TypeVar("TConfig", bound="HMAConfig")
+
+dynamodb = boto3.resource("dynamodb")
```

Comment:
This line is failing during test because there's no region.

One possible fix is to turn this is a standard "how to hide module-level side-effects" and convert it to

def get_dynamodb():
  return boto3.resource("dynamodb")